modules/cnf-sample-single-numa-policy-from-pp.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-numa-aware-scheduling.adoc
+
+:_module-type: REFERENCE
+[id="cnf-sample-single-numa-policy_{context}"]
+= Sample single NUMA node policy
+
+An example performance profile created using the PPC tool is shown here: 
```

Comment:
An example _of_ performance profile ..

Reply:
rephrased it slightly to be in active voice